catboost/python-package/ut/medium/test.py

```diff
@@ -1681,6 +1681,25 @@ def test_cv_custom_loss(task_type):
     assert "test-AUC-mean" in results
     return local_canonical_file(remove_time_from_json(JSON_LOG_PATH))
 
+def test_cv_skip_train(task_type):
```

Comment:
Please also add test with AUC which has default skip_train=True and check that there is no train in results.